firrtl/src/test/scala/firrtlTests/SerializerSpec.scala

```diff
@@ -167,6 +167,37 @@ class SerializerSpec extends AnyFlatSpec with Matchers {
     serialized should be(childModuleTabbed)
   }
 
+  it should "support emitting const types" in {
+    val constInt = DefWire(NoInfo, "constInt", ConstType(UIntType(IntWidth(3))))
+    Serializer.serialize(constInt) should be("wire constInt : const UInt<3>")
+
+    val constAsyncReset = DefWire(NoInfo, "constAsyncReset", ConstType(AsyncResetType))
+    Serializer.serialize(constAsyncReset) should be("wire constAsyncReset : const AsyncReset")
+
+    val constInput = Port(NoInfo, "in", Input, ConstType(SIntType(IntWidth(8))))
+    Serializer.serialize(constInput) should be("input in : const SInt<8>")
+
+    val constBundle = DefWire(
+      NoInfo,
+      "constBundle",
+      ConstType(
+        BundleType(
+          Seq(
+            Field("real", Default, UIntType(IntWidth(32))),
+            Field("imag", Default, UIntType(IntWidth(32))),
+            Field("other", Default, ConstType(SIntType(IntWidth(1))))
```

Comment:
Mega-nit: using both `real` and `imag` is overkill. Just one non-const and one const is good.